internal/development.md

```
@@ -159,16 +159,54 @@ Time: 76.582520
 ```
 Note: Both image and video understanding support streaming and thinking modes as well.
 
-### 6.8. Generate Embeddings
+### 6.8. Chat with OpenAI compatible API
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)